samples/cext-trey-research-python/teamsapp.local.yml

```
@@ -0,0 +1,68 @@
+# yaml-language-server: $schema=https://aka.ms/teams-toolkit/v1.2/yaml.schema.json
+# Visit https://aka.ms/teamsfx-v5.0-guide for details on this file
+# Visit https://aka.ms/teamsfx-actions for details on actions
+version: v1.2
+
+additionalMetadata:
+  sampleTag: Copilot-for-M365-Samples:cext-trey-research-python
+
+provision:
+  # Creates a Teams app
+  - uses: teamsApp/create
+    with:
+      # Teams app name
+      name: cexttreyresearch${{APP_NAME_SUFFIX}}
+    # Write the information of created resources into environment file for
+    # the specified environment variable(s).
+    writeToEnvironmentFile: 
+      teamsAppId: TEAMS_APP_ID
+
+  - uses: script 
+    with:
+      run:
+        echo "::set-teamsfx-env SECRET_STORAGE_ACCOUNT_CONNECTION_STRING=UseDevelopmentStorage=true >> ./env/.env.local.user
```

The echo command is missing the closing quote and so throws an error.

```suggestion
        echo "::set-teamsfx-env SECRET_STORAGE_ACCOUNT_CONNECTION_STRING=UseDevelopmentStorage=true" >> ./env/.env.local.user
```

---

Fixed